atom/browser/api/atom_api_web_contents.cc

```diff
@@ -309,13 +311,23 @@ void WebContents::DidStopLoading(content::RenderViewHost* render_view_host) {
 
 void WebContents::DidGetResourceResponseStart(
     const content::ResourceRequestDetails& details) {
+  net::HttpResponseHeaders* headers = details.headers.get();
+  base::DictionaryValue response_headers;
```

Comment:
We can save the headers in `mate::Dictionary` so there is no need for converting `base::DictionaryValue` to `v8::Object`.